Sources/Workspace/Destination.swift

```diff
@@ -183,3 +181,19 @@ extension Destination: JSONMappable {
         )
     }
 }
+
+private var _hostTriple: Triple?
+extension Triple {
+    public static var hostTriple: Triple {
```

Comment:
Hmm, I don't like this. What if we make triple optional in Destination and absence would imply that it's the host triple (or we rename it to overridenTriple or something). I don't really see any meaningful clients of Destination.triple , it's mostly used in tests but we can just use the toolchain there instead of Destination.

Reply:
Yeah, I'm not a big fan of this. If we remove it, how do you suggest we get access to the default target triple for each platform in `BuildTests`?

Reply:
We can call and cache the result of `getHostTriple(usingSwiftCompiler:)` inside tests.